<?xml version="1.0" ?>
<robot name="vdcar" xmlns:xacro="http://ros.org/wiki/xacro">
  <xacro:include filename="$(find learning_veddha)/urdf/common_properties.xacro"/>

  <xacro:property name="r200_cam_rgb_px" value="0.005"/>
  <xacro:property name="r200_cam_rgb_py" value="0.018"/>
  <xacro:property name="r200_cam_rgb_pz" value="0.013"/>
  <xacro:property name="r200_cam_depth_offset" value="0.01"/>

  <link name="base_footprint"/>

  <joint name="base_joint" type="fixed">
    <parent link="base_footprint"/>
    <child link="base_link" />
    <origin xyz="0 0 0.12065" rpy="0 0 1.57"/>
  </joint>

  <link name="base_link" />
  
<!--unsure-->
  <joint name="imu_joint" type="fixed">
    <parent link="base_link"/>
    <child link="imu_link"/>
    <origin xyz="0.110858 -0.025244 0.238312" rpy="0 0 -1.57"/>
    <axis xyz="0 0 1" />
  </joint>

  <link name="imu_link"/>
<!--unsure-->  
<!--unsure-->
  <joint name="gps_joint" type="fixed">
    <parent link="base_link"/>
    <child link="gps_link"/>
    <origin xyz="0.118658 -0.003244 0.238312" rpy="0 0 0"/>
  </joint>

  <link name="gps_link"/>
<!--unsure-->

  <joint name="rslidar_joint" type="fixed">
    <parent link="base_link"/>
    <child link="rslidar"/>
    <origin xyz="0.7705 0 0.278" rpy="0 0.08726 0"/>
    <axis xyz="0 0 1" />
  </joint>

  <link name="rslidar"/>

  <joint name="laser_joint" type="fixed">
    <parent link="base_link"/>
    <child link="laser"/>
    <origin xyz="-0.063219 0 0.278" rpy="0 0 3.1415926"/>
    <axis xyz="0 0 1" />
  </joint>

  <link name="laser"/>
  
</robot>
